src/libraries/System.Drawing.Primitives/src/System/Drawing/Color.cs

```diff
@@ -487,12 +487,49 @@ private void GetRgbValues(out int r, out int g, out int b)
             b = (int)(value & ARGBBlueMask) >> ARGBBlueShift;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private void MinMaxRgb(out int min, out int max, int r, int g, int b)
+        {
+            if (b < g)
+            {
+                if (b < r)
+                {
+                    min = b;
+                    if (r < g)
+                        max = g;
+                    else
+                        max = r;
```

Comment:
We could simplify this to use `?` assignment?
```suggestion
                    max = r < g ? g : r;
```

Reply:
Thanks, updated.